Tests/NIOTests/DatagramChannelTests.swift

```diff
@@ -106,15 +106,24 @@ final class DatagramChannelTests: XCTestCase {
     private var firstChannel: Channel! = nil
     private var secondChannel: Channel! = nil
 
-    private func buildChannel(group: EventLoopGroup) throws -> Channel {
+    private func buildChannel(group: EventLoopGroup, host: String = "127.0.0.1") throws -> Channel {
         return try DatagramBootstrap(group: group)
             .channelOption(ChannelOptions.socketOption(.so_reuseaddr), value: 1)
             .channelInitializer { channel in
                 channel.pipeline.addHandler(DatagramReadRecorder<ByteBuffer>(), name: "ByteReadRecorder")
             }
-            .bind(host: "127.0.0.1", port: 0)
+            .bind(host: host, port: 0)
             .wait()
     }
+    
+    private var supportsIPv6: Bool {
+        do {
+            let ipv6Loopback = try SocketAddress.makeAddressResolvingHost("::1", port: 0)
+            return try System.enumerateInterfaces().filter { $0.address == ipv6Loopback }.first != nil
```

Comment:
May be better spelled as `.contains(where:)` instead of `filter.first`.

Reply:
Ha - I copied that code from elsewhere in the code base :-)